assets/queries/k8s/allow_privilege_escalation/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "allow_privilege_escalation",
+  "queryName": "Allow Privilege Escalation",
+  "severity": "HIGH",
+  "category": null,
+  "descriptionText": "Containers should not run with allowPrivilegeEscalation in order to prevent gain more privileges than their parent process",
```

Comment:
"Containers should not run with allowPrivilegeEscalation in order to prevent them from gaining more privileges than their parent process"